### DISCUSSION TITLE
docs/v1.10.x/start/sealos-install/

### DISCUSSION THREAD

**Discussion by giscus** [kubeovn/kube-ovn#1963] [Announcements]
# docs/v1.10.x/start/sealos-install/

https://kubeovn.github.io/docs/v1.10.x/start/sealos-install/

**Comment by CraneRZz** [comment]
请问，如果我想用sealos部署Kube-OVN时开启流量镜像功能，应该怎样去配置呢？我试了一下，默认是没有开启流量镜像功能的

**Reply by oilbeater** [reply]
参考 https://kubeovn.github.io/docs/v1.10.x/guide/mirror/ 进行配置就可以

**Comment by shinvdu** [comment]
不行， 本地和linode报一样的错

root@localhost:~/k8s#  sealos run labring/kubernetes:v1.24.3 labring/kube-ovn:v1.10.5  \
   --masters 192.168.193.67 \
   --nodes 192.168.200.52 -p 'GmC%FFZ0$K6Ng@mEOu'  
2023-11-22T13:19:38 info sync new version copy pki config: /var/lib/sealos/data/default/pki /root/.sealos/default/pki
2023-11-22T13:19:38 info sync new version copy etc config: /var/lib/sealos/data/default/etc /root/.sealos/default/etc
2023-11-22T13:19:38 info start to install app in this cluster
are you sure to override these following apps? 
labring/kubernetes:v1.24.3
labring/kube-ovn:v1.10.5
 
 	 Yes [y/yes], No [n/no] : y 
Resolving "labring/kubernetes" using unqualified-search registries (/etc/containers/registries.conf)
Trying to pull docker.io/labring/kubernetes:v1.24.3...
Getting image source signatures
Copying blob 2aec564fb600 done  
Copying blob 4be646a2cb5d done  
Copying blob 432dbf7a32b9 done  
Copying blob e761fc4568b1 done  
Copying config 5ff932387d done  
Writing manifest to image destination
Storing signatures
Resolving "labring/kube-ovn" using unqualified-search registries (/etc/containers/registries.conf)
Trying to pull docker.io/labring/kube-ovn:v1.10.5...
Getting image source signatures
Copying blob d1cf0cd1700f done  
Copying config 6469502337 done  
Writing manifest to image destination
Storing signatures
2023-11-22T13:20:05 error Applied to cluster error: invalid reference format
Error: invalid reference format
invalid reference format

**Comment by liu9969921666** [comment]
有没有部署纯IPV6的容器集群方案
